app/pyalbert_utils.py

```diff
@@ -35,7 +35,7 @@ def new_chat(config: Config) -> int:
     return chat_id
 
 
-def generate(config: Config, query: str) -> str | None:
```

Comment:
@dtrckd Same. This raises a Mypy error if it doesn't take into account the None case.

Reply:
the generate function never returns None ?

Reply:
The generate function can returns nothing: https://github.com/etalab-ia/albert-tchapbot/blob/dev/app/pyalbert_utils.py#L60

Reply:
Btw I suggest to use Mypy or a IDE type checker to check for that, type hints become really useful when it is checked in real time

Reply:
Can I revert your commit?